tests/ccl_test_pyccl_interface.py

```diff
@@ -364,13 +406,30 @@ def check_corr(cosmo):
     lens1 = ccl.ClTracerLensing(cosmo, False, n=n, z=z)
     lens2 = ccl.ClTracerLensing(cosmo, True, n=(z,n), bias_ia=(z,n), f_red=(z,n))
 
-    ells=np.arange(3000)
-    cls=ccl.angular_cl(cosmo,lens1,lens2,ells)
-
-    t=np.logspace(-2,np.log10(5.),20) #degrees
-    corrfunc=ccl.correlation(cosmo,ells,cls,t,corr_type='L+',method='FFTLog')
-    assert_( all_finite(corrfunc))
-
+    ells = np.arange(3000)
+    cls = ccl.angular_cl(cosmo, lens1, lens2, ells)
+
+    t_arr = np.logspace(-2., np.log10(5.), 20) # degrees
+    t_lst = [t for t in t_arr]
```

Comment:
what about using `t_lst = t_arr.tolist()`?

Reply:
That's a good tip, but I don't think it matters too much. Let's leave this unchanged.

Reply:
Yeah, here it doesn't make any difference.